gsa/src/gmp/commands/alerts.js

```diff
@@ -186,19 +186,17 @@ class AlertCommand extends EntityCommand {
       const {new_alert} = response.data;
       new_alert.report_formats = map(
         new_alert.get_report_formats_response.report_format,
-        format => new Model(format),
+        format => Model.fromElement(format),
```

Comment:
import {parseModelFromElement} from 'gmp/model';
```suggestion
        format => parseModelFromElement(format, 'reportformat'),,
```